docs/content/users/debugging-profiling/step-debugging.md

```diff
@@ -87,6 +88,25 @@ However, if you’ve not yet used PhpStorm with Xdebug for a regular web request
 !!!tip "If you’re using VS Code on Windows with WSL2"
     VS Code should suggest two extensions if you have WSL2 enabled along with a PHP project: “PHP Debug” and “Remote - WSL”. You’ll need to enable both of these extensions in your distro (e.g. Ubuntu).
 
+
+### Netbeans Debugging Setup
+
+The Apache Netbeans IDE is configured by default to accomodate Drupal and Symfony projects.
+
+Complete the following DDEV configuration steps to connect the Netbeans debugger to XDEBUG:
+
+1. Create a project in Netbeans and set the project URL to use DDEV's project URL. For example use `https://drupal.ddev.site` and specify the index file as index.php. ![debugger-settings-screenshot](../../images/netbeans18-project-configuration.png)
+2. Create the xdebug.ini file in `.ddev/php/` directory. The full path will be: `.ddev/php/xdebug.ini`
+3. Add the PHP configuration to the xdebug.ini file:
+```ini
+[PHP]
+xdebug.mode=debug
+xdebug.idekey="netbeans-xdebug"
+```
+4. Restart ddev: `ddev restart`
+5. Verify that Netbeans is configured as shown in the screenshot ![project-properties-screenshot](../../images/netbeans18-configuration.png)
+6. For debugging Use the command `telnet localhost 9003` to verify that the xdebug is available. To exit telnet press `control+]`and then enter the `quit` command.
```

Comment:
It's not clear why this is necessary. And `telnet` isn't on everybody's host system.

Reply:
Yes, we I think we can remove the debugging step.